contrib/linux/dosbox-staging.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=dosbox-staging
+GenericName=DOS emulator
+Comment=A modernized DOSBox
+Exec=dosbox
+Icon=dosbox-staging
+Type=Application
+Keywords=DOS,gaming,games,game,emulator
```

Comment:
`.desktop` entries use `;` for a separator, and expect a trailing `;` in lists, so rewrite this line as:

    Keywords=dos;gaming;game;games;emulator;

and a line to specify if program should be started in the terminal or not:

    Terminal=false